### PR TITLE
Updating Terraform config example for GKE.

### PR DESCRIPTION
**Pull request by soumyapani**

### CODE REVIEW

File: aiinfra-cluster/variables.tf [hunk 1]
```diff
@@ -72,6 +73,7 @@ variable "instance_count" {
 variable "accelerator_type" {
   description = "The accelerator (GPU) type."
   type        = string
+  default     = "nvidia-tesla-a100"
```
Comment (stevenBorisko) [inline aiinfra-cluster/variables.tf:76]: does this work for migs which I thought from [this](https://cloud.google.com/compute/docs/gpus#a100-gpus) requires the machine type to be an a2 if the desired gpu is an a100?

File: aiinfra-cluster/variables.tf [hunk 2]
```diff
@@ -87,6 +89,11 @@ variable "instance_image" {
     family  = string,
     project = string
   })
+  default = {
+    family  = "tf-latest-gpu-gvnic-debian-10"
+    name    = null
+    project = "deeplearning-platform-release"
+  }
```
Comment (stevenBorisko) [inline aiinfra-cluster/variables.tf:96]: should this go with the slurm default [here](https://github.com/GoogleCloudPlatform/ai-infra-cluster-provisioning/blob/develop/aiinfra-cluster/main.tf#L92)?

Comment (soumyapani) [reply aiinfra-cluster/variables.tf]: that is correct. Updated in latest iteration.

File: aiinfra-cluster/variables.tf (hunk 1 above)
Comment (soumyapani) [reply aiinfra-cluster/variables.tf:76]: default GPU count is set to 0. So it works as is. But if the user changes the GPU count, they have to choose the right vm type. Hopefully we will get to determine the VM type ourselves based on number of GPUs asked.

**Review by sdlin — APPROVED** [review]

**Review by stevenBorisko — APPROVED** [review]

**Pull request merged**